cdhweb/people/views.py

```diff
@@ -37,6 +37,16 @@ def get_past(self):
         current = self.get_current()
         return self.object_list.exclude(id__in=current.values('id'))
 
+    def grant_label(self, grant):
+        """Convert a grant into a label with date."""
+        # if a fellowship, display as "X fellow"
+        if "Fellow" in grant.grant_type.grant_type:
+            no_ship = grant.grant_type.grant_type.split('ship', 1)[0]
+            return f"{grant.years} {no_ship}"
```

Comment:
😆  🚫 🚢

Reply:
I might consolidate into a single return since the grant years + label is consistent for both cases, but I don't feel strongly about that. codeclimate or one of the other checkers might complain about an else after a return (unnecessary)